gtk/src/auto/container.rs

```diff
@@ -175,6 +186,19 @@ impl<O: IsA<Container>> ContainerExt for O {
     //    unsafe { TODO: call ffi:gtk_container_child_get() }
     //}
 
+    fn child_get_property(&self, child: &impl IsA<Widget>, property_name: &str) -> glib::Value {
```

Comment:
Maybe we can make the gir produces code that make use of this once merged instead of going through ffi as well

Reply:
Yes please. Just like your PR for normal properties :)